redash/destinations/pagerduty.py

```diff
@@ -0,0 +1,69 @@
+import pypd
+import logging
+
+from redash.destinations import *
+
+
+class Pagerduty(BaseDestination):
+
+    KEY_STRING = '{user_name}_{user_email}_{query_id}_{query_name}'
+    DESCRIPTION_STR = 'redash_{query_id}_{query_name}'
```

Comment:
It's a string for humans not 🤖. I think `Redash Query #{query_id}: {query_name}` is a much friendlier default :)